examples/axisartist/demo_parasite_axes2.py

```diff
@@ -1,27 +1,26 @@
 """
-===================
-Demo Parasite Axes2
-===================
-
+==================
 Parasite axis demo
+==================
 
-The following code is an example of a parasite axis. It aims to show how
-to plot multiple different values onto one single plot. Notice how in this
-example, par1 and par2 are both calling twinx meaning both are tied directly to
-the x-axis. From there, each of those two axis can behave separately from the
-each other, meaning they can take on separate values from themselves as well as
-the x-axis.
+The following code is an example of a parasite axis. It shows how to plot
+multiple different values onto one single plot. Notice how in this example,
+*par1* and *par2* are both obtained by calling twinx, meaning both are
+tied directly to the x-axis. From there, each of those two axis can behave
+separately from the each other, meaning they can take on separate values from
```

Comment:
```suggestion
separately from each other, meaning they can take on separate values from
```

Reply:
I ended up rewriting the whole paragraph.